totest-manager.py

```diff
@@ -410,11 +403,8 @@ class ToTestFactoryPowerPC(ToTestBase):
     def __init__(self, project, dryrun):
         ToTestBase.__init__(self, project, dryrun)
 
-    def openqa_version(self):
-        return 'Tumbleweed'
-
     def openqa_group(self):
-        return 'PowerPC'
+        return 'Tumbleweed PowerPC'
```

Comment:
isn't it 'openSUSE Tumbleweed PowerPC'? I thought that is the correct group name we had for powerpc